src/transformState.js

```diff
@@ -59,10 +59,34 @@
  * { foo: 'bar', name: 'Jim' }
  *
  * @param {Object} state
- * @param {Object[]} transforms
+ * @param {Object[ ]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const { operation, properties } of transforms) {
+    switch (operation) {
+      case 'addProperties': {
+        Object.assign(state, properties);
+
+        break;
+      }
+
+      case 'removeProperties': {
+        for (const i of properties) {
+          delete state[i];
```

Comment:
Краще для розуміння замінити і на prop, key:
```suggestion
        for (const property of properties) {
          delete state[property];
```

Reply:
Так